docs/config/cluster.rst

```diff
@@ -30,6 +30,32 @@ in the configuration of each node in the cluster.
    exactly the same on every node in the cluster for proper operation of the
    cluster.
 
+External Authentication
+-----------------------
+
+.. _external_auth:
+
+Default settings for the :ref:`JWT authentication <auth_jwt>`.
+Defined in the ``auth.host_based.jwt`` setting in
+the ``crate.yml`` file.
+If JWT properties are not provided by the :ref:`CREATE USER <create-user-jwt>`
+statement, default values are used.
+CrateDB username is dictated by the 'username' property of a JWT token.
+
+.. _auth.host_based.jwt.iss:
+
+**auth.host_based.jwt.iss**
+  | *Runtime:* ``no``
+
+  Default value for the ``iss`` `:ref:`JWT property <create-user-jwt>`.
+
+.. _auth.host_based.jwt.aud:
+
+**auth.host_based.jwt.aud**
+  | *Runtime:* ``no``
+
+  Default value for the ``aud`` :ref:`JWT property <create-user-jwt>`.
+
```

Comment:
```suggestion
```

We don't mention auth/hba here. This needs to go into the hba.rst docs

Reply:
But then we end up with not having some cluster settings in the cluster settings page, is it okay? 

For example, we mention some hba settings in the hba.rst but actual values are described in the node.rst 

See reference in the hba.rst:
> The description of these fields can be found in :ref:`host_based_auth`.


I was following the same logic.

Reply:
> But then we end up with not having some cluster settings in the cluster settings page, is it okay?
> For example, we mention some hba settings in the hba.rst but actual values are described in the node.rst

Oh right. But then it needs to go into `node.rst`, not `cluster.rst`, together with the other hba values.

Reply:
>But then it needs to go into node.rst

Other hba settings are specifically node-level, quote from node.rst:

> Authentication settings (``auth.host_based.*``) are node settings, which means
that their values apply only to the node where they are applied and different
nodes may have different authentication settings.

https://github.com/crate/crate/issues/16597 says global config is cluster level. But more imprtantly, I don't think it makes sense to have different `iss` per node - we can end up with different auth provider, depending on the handler node?

Reply:
> But more imprtantly, I don't think it makes sense to have different iss per node - we can end up with different auth provider, depending on the handler node?

Yes - same as you can have different hba configurations.
Maybe you've region local providers or something. Not that I suspect that to be common, but I don't think it warrants splitting the hba settings into cluster and node. 
Makes it harder to get a complete overview if the docs for it are scattered.